toolkit/tools/graphpkgfetcher/graphpkgfetcher.go

```diff
@@ -162,8 +176,7 @@ func resolveGraphNodes(dependencyGraph *pkggraph.PkgGraph, inputSummaryFile, out
 
 // resolveSingleNode caches the RPM for a single node.
 // It will modify fetchedPackages on a successful package clone.
-func resolveSingleNode(cloner *rpmrepocloner.RpmRepoCloner, node *pkggraph.PkgNode, toolchainPackages []string, fetchedPackages, prebuiltPackages map[string]bool, outDir string) (err error) {
-	const cloneDeps = true
+func resolveSingleNode(cloner *rpmrepocloner.RpmRepoCloner, node *pkggraph.PkgNode, cloneDeps bool, toolchainPackages []string, fetchedPackages, prebuiltPackages map[string]bool, outDir string) (err error) {
```

Comment:
>cloneDeps [](http://example.com/codeflow?start=84&length=9)

Just checking: we moved `cloneDeps` into an argument as preparation for the delta build stuff?

Reply:
Yes, we need the updated tdnf as well, but it doesn't make sense for the delta fetcher to pull runtime dependencies, we only want a single specific .rpm file.

Reply:
(current tdnf breaks if we set deps=false)